### DISCUSSION TITLE
Gruntwork Pipelines and Terraform Cloud

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#451] [Help] [answered]
I have 50 accounts that Terraform Cloud Workspaces make it tricky to promote changes in an automated way. I cannot auto-approve changes, but this means I have to deploy 50 workspaces individually. 

How can I adopt Gruntwork Pipelines with Terraform Cloud?

---

<ins datetime="2022-06-09T14:06:47Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108733">Tracked in ticket #108733</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
Gruntwork Pipelines does not natively support Terraform Cloud at the moment. However, you can make it work with Terraform Cloud by adopting the method specified [in this blog post](https://blog.gruntwork.io/how-deploy-production-grade-infrastructure-using-gruntwork-with-terraform-cloud-aca919ca92c2). Once you have `terragrunt` working with Terraform Cloud, `apply` and `plan` calls from Gruntwork Pipelines will automatically translate to Terraform Cloud actions.
